src/test/java/firstSimpleJavaTasks/ArrayEditingTests.java

```diff
@@ -0,0 +1,39 @@
+package firstSimpleJavaTasks;
+
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class ArrayEditingTests {
+
+    private int[] inputArray;
+    private int[] editedArray;
+    private int[] numberForRemoval;
+
+    public ArrayEditingTests (int[] inputArray, int[] editedArray, int[] numberForRemoval) {
+        this.inputArray = inputArray;
+        this.editedArray = editedArray;
+        this.numberForRemoval = numberForRemoval;
+    }
+
+//name = "{index}:{inputArray}, {editedArray}, {numberForRemoval}"
+    @Parameterized.Parameters()
+    public static Collection dataForTest() {
+        return Arrays.asList(new Integer[][][] {
```

Comment:
You can use arrays of objects for returning from data providers 
For example: new Object[][] {{ new int[]{....}, new int[]{...}, 0 }}

Reply:
Type for returning dataset was changed to Object[][]